pkg/products/monitoring/prometheusRules.go

```diff
@@ -99,15 +99,14 @@ func (r *Reconciler) reconcileKubeStateMetricsAlerts(ctx context.Context, server
 			Namespace: r.Config.GetOperatorNamespace(),
 		},
 	}
-
 	rules := []monitoringv1.Rule{
 		{
 			Alert: "KubePodCrashLooping",
 			Annotations: map[string]string{
 				"sop_url": "https://github.com/RHCloudServices/integreatly-help/blob/master/sops/alerts_and_troubleshooting.md",
 				"message": "Pod {{  $labels.namespace  }} / {{  $labels.pod  }} ({{  $labels.container  }}) is restarting {{  $value  }} times every 5 minutes; for the last 15 minutes",
 			},
-			Expr:   intstr.FromString("rate(kube_pod_container_status_restarts_total{job='kube-state-metrics'}[15m]) * on (namespace, namespace) group_left(label_monitoring_key) kube_namespace_labels{label_monitoring_key='middleware'} * 60 * 5 > 0"),
+			Expr:   intstr.FromString("rate(kube_pod_container_status_restarts_total{job='kube-state-metrics'}[10m]) * on (namespace, namespace) group_left(label_monitoring_key) kube_namespace_labels{label_monitoring_key='middleware'} * 60 * 5 > 0"),
```

Comment:
Would it be better to lower the `for` threshold to 10m, and set this range to `[5m]` to match the multiplier at the end?

Reply:
I'll test that out :+1:

Reply:
Seems like it's also never reaching the firing state with `[5m]` and `for:10`, keeps alternating between green and pending :thinking:  so it's the same problem that was occuring with `[5m]` and `for:15`, going to test it once more with the current change

Reply:
@david-martin Out of curiosity, what does the right side exactly mean `* 60 * 5 ` ?

Reply:
@pepedocs `kube_pod_container_status_restarts_total{}[10m]` gives the rate of restarts per second over a 10m range. The `* 60 * 5` is to scale that figure up to get a more user friendly number for the alert message i.e. `restarting {{  $value  }} times every 5 minutes `

@HubertStefanski Can you update the message text as it has `15 minutes` hardcoded